README.md

```diff
@@ -31,24 +31,11 @@ If you need any help or want to discuss with other F3D users and developers, hea
 
 # Quickstart
 
-Open a file and visualize it interactively:
+Open a file directly in F3D <u>OR</u> from the command line by running `f3d /path/to/file.ext`. Optionally, append `--output=/path/to/img.png` to save the rendering into an image file.
 
-```
-f3d /path/to/file.ext
-```
+Get help by running `f3d --help` or `man f3d` (Linux only).
 
-Open a file and save the rendering into an image file:
-
-```
-f3d /path/to/file.ext --output=/path/to/img.png
-```
-
-Get help:
-
-```
-f3d --help
-man f3d # Linux only
-```
+See the [Quickstart Guide](QUICKSTART.md) for more information about getting started with F3D.
```

Comment:
Im not sure I prefer this version tbh. The info is the same, just hard to get.


How about keeping just the first command in ``` ?

Reply:
do you think we should keep the second command then?

also do you think we should expand the section slightly with 1-2 basic key bindings? or leave it more concise?

Reply:
I think we should keep a single command here.

Since we now have a quick start guide, lets keep this section super short, so id say no bindings here.